services/data_plane_manager/pom.xml

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.2.6.RELEASE</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+    <groupId>com.futurewei.alcor</groupId>
+    <artifactId>AlcorDataPlaneManager</artifactId>
+    <version>0.1.0-SNAPSHOT</version>
+    <name>AlcorDataPlaneManager</name>
+    <description>Alcor Data Plane Manager Module</description>
+
+    <properties>
+        <java.version>1.8</java.version>
```

Comment:
I think other projects have been upgrade to JDK 11. Can we change that accordingly in this pom?

Also, see if we can remove any dependencies or plugins to simplify it?

Reply:
fixed in 55a056a